src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -370,6 +406,35 @@ protected static String getNameIndex(String topic, int partition) {
     return topic + "_" + partition;
   }
 
+  public void configureSingleTableLoadFromMultipleTopcis(boolean fixEnabled) {
+    enableStageFilePrefixExtension = fixEnabled;
+  }
+
+  /**
+   * util method, checks if there are stage files present matching "old" file name format, if they
```

Comment:
Nit: can we be more specific than just saying "old"? After some time, it would be hard to understand.